jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Retainable.java

```diff
@@ -13,7 +13,139 @@
 
 package org.eclipse.jetty.util;
```

Comment:
should this be in jetty-io?

Reply:
Was originally (in Jetty 10) in `util`, but I agree it should be in `io`.